.nycrc

```diff
@@ -0,0 +1,12 @@
+{
+  "check-coverage": true,
+  "lines": 75,
+  "statements": 80,
+  "functions": 75,
+  "branches": 75,
+  "temp-directory": "coverage",
+  "reporter": [
+    "text",
+    "html"
+  ]
+}
```

Comment:
Have you considered doing the same. as in [plugin-lightning-dev](https://github.com/salesforcecli/plugin-lightning-dev/blob/main/.nycrc)?

Reply:
That is no good. In fact, plugin-lightning-dev should be updated IMHO. If we just extend from `@salesforce/dev-config/nyc` then coverage limits are set to 50% which is too low